### PR TITLE
[ADMIN][ADI] Add idea template admin

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/EventListener/FilterConfiguratorListener.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+use Sonata\AdminBundle\Controller\CoreController;
+use Sonata\AdminBundle\Controller\CRUDController;
+use Sonata\AdminBundle\Controller\HelperController;
+use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+
+class FilterConfiguratorListener
+{
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
```
Comment (Remg) [inline src/EventListener/FilterConfiguratorListener.php]: you need to declare `em` as a property

File: src/Admin/Extension/FilterConfigurationAdminExtension.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace AppBundle\Admin\Extension;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
+use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+
+class FilterConfigurationAdminExtension extends AbstractAdminExtension
```
Comment (ottaviano) [inline src/Admin/Extension/FilterConfigurationAdminExtension.php]: what about prefix by `DoctrineFilter...`?

File: src/Admin/IdeasWorkshopQuestionAdmin.php [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+class IdeasWorkshopQuestionAdmin extends AbstractAdmin
+{
+    public function createQuery($context = 'list')
+    {
+        /** @var QueryBuilder $proxyQuery */
+        $proxyQuery = parent::createQuery($context);
+
+        $proxyQuery
+            ->leftJoin('o.guideline', 'g')
+            ->orderBy('g.position', 'ASC')
```
Comment (ottaviano) [inline src/Admin/IdeasWorkshopQuestionAdmin.php]: It seems that doing so will disable other sort possibilities (by name or category) on the list page.

File: src/Admin/IdeasWorkshopQuestionAdmin.php [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+            ])
+            ->add('enabled', null, [
+                'label' => 'Visibilité',
+            ])
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
```
Comment (ottaviano) [inline src/Admin/IdeasWorkshopQuestionAdmin.php]: what about reusing here you QuestionType?
ex:
```
    $builder = $formMapper
        ->getFormBuilder()
        ->getFormFactory()
        ->createBuilder(QuestionType::class);

        $formMapper
            ->add($builder->get('category'))
            ->add($builder->get('name'))
            // ...
        ;
```

https://sonata-project.org/bundles/admin/3-x/doc/reference/form_types.html#adding-a-formbuilderinterface

File: src/Form/IdeasWorkshop/QuestionType.php [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+                'data_class' => Question::class,
+            ])
+        ;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('category', TextType::class, [
+                'label' => 'Titre de question',
```
Comment (ottaviano) [inline src/Form/IdeasWorkshop/QuestionType.php]: is it the good label text?

File: src/Admin/IdeasWorkshopGuidelineAdmin.php [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+            ->add('position', null, [
+                'label' => 'Position',
+            ])
+            ->add('enabled', null, [
+                'label' => 'Visibilité',
+            ])
+            ->add('questions', CollectionType::class, [
+                'label' => 'Questions',
+                'entry_type' => QuestionType::class,
+                'prototype' => true,
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply src/Admin/IdeasWorkshopGuidelineAdmin.php]: Removed.

File: src/Admin/Extension/FilterConfigurationAdminExtension.php (hunk 2 above)
Comment (illusionOfParadise) [reply src/Admin/Extension/FilterConfigurationAdminExtension.php]: Done.

**Pull request merged**